windows/unicode.mpl

```diff
@@ -9,39 +9,40 @@
 "String.makeStringView" use
 "control.Nat16"         use
 "control.assert"        use
+"control.keep"          use
+"control.swap"          use
 
 "kernel32.CP_UTF8"              use
 "kernel32.MB_ERR_INVALID_CHARS" use
 "kernel32.MultiByteToWideChar"  use
 
 utf16: [
   source: makeStringView;
-  destSize:
+  targetNat16Count:
     0
     0nx
     source.size
     source.data storageAddress
     MB_ERR_INVALID_CHARS
-    CP_UTF8 MultiByteToWideChar
+    CP_UTF8
+    MultiByteToWideChar
   ;
```

Comment:
I am unsure if a semicolon should be placed on a separate line. The only places in sl where this is the case are here and in Process.mpl.

Reply:
https://github.com/Matway/mpl-sl/blob/9d5b602d112bd3cf1ea0d328ef7d1c212631cc12/Union.mpl#L38-L48